src/main/java/org/geysermc/discordbot/commands/FloodgateUuidCommand.java

```diff
@@ -77,25 +76,29 @@ protected void execute(SlashCommandEvent event) {
         }
     }
 
+    /**
+     * @param username Bedrock username
+     * @param uuid The converted xuid "floodgate uuid"
+     * @param error Error occurs when floodgate API is offline.
+     * @return Returns embed that contains floodgate uuid or state Floodgate API.
+     */
     private MessageEmbed floodgateUUID(String username, UUID uuid, boolean error) {
         EmbedBuilder builder = new EmbedBuilder();
         builder.setTitle("Floodgate UUID");
 
-        if (!error) {
-            if (uuid != null) {
-                builder.addField("PlayerName", username, false);
-                builder.addField("Floodgate UUID", uuid.toString(), false);
-                builder.setColor(BotColors.SUCCESS.getColor());
-            } else {
-                builder.addField("Error", "Could not find bedrock player: " + username, false);
-                builder.setColor(BotColors.FAILURE.getColor());
-            }
-        } else {
+        if (error) {
             builder.addField("Error", "Unable to lookup uuid, FloodgateAPI currently unavailable", false);
             builder.setColor(BotColors.FAILURE.getColor());
+            return builder.build();
```

Comment:
Can we change this back to how it was before the last commit so there are not a tonne of `builder.build` calls as I feel it will read better?
Please voice your reasoning for doing it as it may make more sense.